syntax = "proto3";

message RequestMsg
{
	int32 cmdType = 1;
	bytes clientID = 2;
	bytes serverID = 3;
	string sign = 4;
	string data = 5;
}
